### PR TITLE
Updated details for AG recommendations

### PR DESCRIPTION
**Pull request by joeyouss**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/1-ccm-recommendations/governance.md [hunk 1]
```diff
@@ -6,74 +6,68 @@ description: Optimize AWS costs with asset governance recommendations
 
 # Optimize AWS costs with asset governance recommendations
```
Comment (iamanmolm) [inline docs/cloud-cost-management/4-use-ccm-cost-optimization/1-ccm-recommendations/governance.md:7]: Optimize AWS/Azure/GCP costs with asset governance recommendations

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/1-ccm-recommendations/governance.md [hunk 2]
```diff
@@ -6,74 +6,68 @@ description: Optimize AWS costs with asset governance recommendations
...
 
-Harness Cloud Asset Governance provides tools to optimize your cloud spend and avoid unnecessary costs. The asset governance recommendations work by rightsizing resources and decommissioning unused instances. By leveraging these recommendations, you can better control your cloud expenses while ensuring that your cloud infrastructure is optimized for maximum efficiency and cost-effectiveness.
+Harness Cloud Asset Governance provides tools to optimize your cloud spend and avoid unnecessary costs. By leveraging these recommendations, you can better control your cloud expenses while ensuring that your cloud infrastructure is optimized for maximum efficiency and cost-effectiveness.
 
-You can view the recommendations for all of your AWS accounts on the recommendations page.
+You can view the recommendations for all of your Cloud accounts on the recommendations page.
 
-   <DocImage path={require('./static/governance.png')} width="50%" height="50%" title="Click to view full size image" />
+   <DocImage path={require('./static/Overviewpg1.png')} width="90%" height="90%" title="Click to view full size image" />
```
Comment (iamanmolm) [inline docs/cloud-cost-management/4-use-ccm-cost-optimization/1-ccm-recommendations/governance.md:13]: In the image few things to be corrected:
1. It is showing ec2 recommendations as well, kindly use filter to get only governance recommendation.
2. "Total Monthly Savings that can be achieved when all open recommendations are applied."
3. "Total Monthly Cost incurred as of today."
4. "Monthly Savings that can be achieved if the recommendation is applied."
5. "Monthly Cost incurred as of today."
6. "Recommended Action for recommendation."
7. 3 dots explanations is missing
8. Add description about Export csv: "Export CSV to download and share your recommendations data effortlessly."
9. Add description about Manage Ignore List: "Manage Ignore List to exclude specific resource from recommendations and re-add them when needed."

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/1-ccm-recommendations/governance.md [hunk 3]
```diff
@@ -6,74 +6,68 @@ description: Optimize AWS costs with asset governance recommendations
...
 
-The following resource types can be utilized optimally by creating governance rules to rightsize the resources:
 
-- EC2 instances
+   <DocImage path={require('./static/Overviewpg2.png')} width="90%" height="90%" title="Click to view full size image" />
```
Comment (iamanmolm) [inline docs/cloud-cost-management/4-use-ccm-cost-optimization/1-ccm-recommendations/governance.md:16]: In the image few things to be corrected:
1. Screenshot should have only governance recommendations
2. "Total Monthly Savings achieved by applying recommendations."
3. "Recommended Action for recommendation."
4. "Recommendation Applied Timestamp."

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/1-ccm-recommendations/governance.md [hunk 4]
```diff
@@ -6,74 +6,68 @@ description: Optimize AWS costs with asset governance recommendations
...
 
-- RDS instances 
+The following resource types can be utilized optimally by creating governance rules:
```
Comment (iamanmolm) [inline docs/cloud-cost-management/4-use-ccm-cost-optimization/1-ccm-recommendations/governance.md:18]: The following resources can be optimized with Asset Governance:

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/1-ccm-recommendations/governance.md [hunk 5]
```diff
@@ -6,74 +6,68 @@ description: Optimize AWS costs with asset governance recommendations
...
-- EBS Volumes 
+### AWS Resource Coverage (Comprehensive list)
 
-:::note
-Before using recommendations in your environment, ensure that you evaluate their impact thoroughly. The person reviewing the recommendations should be able to understand the impacts identified in the recommendations, as well as the impact on the infrastructure and business.
+- EC2 instances
+- S3 buckets
+- Lambda functions
+- RDS (Relational Database Service) instances
+- CloudFormation stacks
```
Comment (iamanmolm) [inline docs/cloud-cost-management/4-use-ccm-cost-optimization/1-ccm-recommendations/governance.md:26]: This doesn't look comprehensive. Kindly check the list from recommendations page.

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/1-ccm-recommendations/governance.md [hunk 6]
```diff
@@ -6,74 +6,68 @@ description: Optimize AWS costs with asset governance recommendations
...
 
-Using recommendations without proper assessment could result in unexpected changes, such as issues with system performance or poor reliability.
-:::
+### Azure Resource Coverage (Comprehensive list)
```
Comment (iamanmolm) [inline docs/cloud-cost-management/4-use-ccm-cost-optimization/1-ccm-recommendations/governance.md:28]: This doesn't look comprehensive. Kindly check the list from recommendations page.

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/1-ccm-recommendations/governance.md [hunk 7]
```diff
@@ -6,74 +6,68 @@ description: Optimize AWS costs with asset governance recommendations
...
+- Storage accounts
+- App services
+- Cosmos DB accounts
+- Key Vaults
 
-* Connect your AWS cloud account in Harness and set up CCM for cost management. For more information, go to [Set up cost visibility for AWS](../../get-started/onboarding-guide/set-up-cost-visibility-for-aws.md).
-* To obtain governance recommendations, configure a Harness AWS CCM connector with the **Cloud Governance** feature enabled.
-* Make sure that you have added the required permissions in your AWS account. Go to [AWS access permissions for Cloud asset governance](../../get-started/onboarding-guide/set-up-cost-visibility-for-aws.md#cloud-asset-governance-rules).
-* To add missing permissions, go to [Add permissions](../../get-started/onboarding-guide/set-up-cost-visibility-for-aws.md#add-permissions).
+### GCP Resource Coverage (Comprehensive list)
```
Comment (iamanmolm) [inline docs/cloud-cost-management/4-use-ccm-cost-optimization/1-ccm-recommendations/governance.md:36]: This doesn't look comprehensive. Kindly check the list from recommendations page.

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/1-ccm-recommendations/governance.md [hunk 8]
```diff
@@ -6,74 +6,68 @@ description: Optimize AWS costs with asset governance recommendations
...
-8. Toggle the **Dry Run** mode if you do not want to take action immediately.
-9. Click **Finish**. 
-   
-If you wish to modify any of these Out-of-the-box rules, select **Open in Rule Editor**.
-  The rule is cloned and opened in a new tab.
+3. There are two tabs present on the window - "Open Recommendations" and "Applied Recommendations". Open Recommendation tab shows all the recommendations that are currently available for applying and the potential monthly savings that the user can achieve if they are applied. On the other hand, Applied Recommendations show all the recommendations that have already been applied and the total savings achieved with their application. 
+4. The recommendations window allows you to put filters on the recommendations to see the result as per convenience. Kindly choose "Governance" in "Recommendation Type" filter for all governance recommendations. Currently, these are filters supported:
+
+  <DocImage path={require('./static/filtersAG.png')} width="60%" height="60%" title="Click to view full size image" />
```
Comment (iamanmolm) [inline docs/cloud-cost-management/4-use-ccm-cost-optimization/1-ccm-recommendations/governance.md:68]: 1. Highlight "Resource Name", use this filter all recommendations for a given Governance Rule Name.
2. Highlight "Namespace / Azure Subscription / AWS Account Name / GCP Project ID", use this filter to get recommendation only for a particular cloud account.
3. Highlight "Recommendation Type", use this to get only Governance Recommendations.

**Review by iamanmolm — CHANGES_REQUESTED** [review]

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/1-ccm-recommendations/governance.md (hunk 1 above)
Comment (joeyouss) [reply docs/cloud-cost-management/4-use-ccm-cost-optimization/1-ccm-recommendations/governance.md:7]: done

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/1-ccm-recommendations/governance.md (hunk 4 above)
Comment (joeyouss) [reply docs/cloud-cost-management/4-use-ccm-cost-optimization/1-ccm-recommendations/governance.md:18]: done

**Review by iamanmolm — DISMISSED** [review]

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/1-ccm-recommendations/static/filter2.png
Comment (iamanmolm) [inline docs/cloud-cost-management/4-use-ccm-cost-optimization/1-ccm-recommendations/static/filter2.png:1]: This image can be deleted.

**Review by iamanmolm — APPROVED** [review]

**Pull request merged**